src/Camera.h

```diff
@@ -59,8 +59,12 @@ class Camera
 	unsigned int pixel_height;
 
 protected:
-        // Perspective settings
+	// Perspective settings
 	double viewer_distance;
 	// Orthographic settings
 	double height; // world-space height of viewport
+
+private:
+	// force angle to be 0-360
+	static double wrap(double angle) { return fmodf(360 + angle, 360); }
```

Comment:
This method doesn't really belong in this class. If you move the implementation of `setVp*()` out of the header file, this could be a local helper function (even in an anonymous namespace to avoid any potential namespace pollution).

Reply:
OK I have moved them but moving these one liners into the cpp file in the usual format adds another 25 lines of code that has to be kept in sync with the header. That is why I have always written one line accessors inline. How does a private function pollute a namespace?

Reply:
Private functions don't pollute the namespace, they just becomes part of the class API, which is unnecessary for such implementation defaults.
For helper functions, you can avoid polluting the namespace by declaring them `static`, or put them in an anonymous namespace, which should be pretty much the same thing. (`static` functions were deprecated for a while during standardization, so I guess a bunch of people adopted anonymous namespaces early on, but I believe the difference just comes down to preference).